epub3-pub-utils/src/main/resources/xml/xproc/create-package-doc.xpl

```diff
@@ -66,7 +66,7 @@
                 </p:inline>
             </p:input>
         </p:uuid>
-        <p:wrap-sequence wrapper="d:wrapper">
+        <p:wrap-sequence wrapper="_">
```

Comment:
I'm not doing anything special here. Just removed the `d:` prefix again (reverted https://github.com/daisy/pipeline-scripts-utils/commit/a5ec2380a7aa3a32c13ce55557f809b7fd6d4227) but instead of calling it `wrapper` I chose `_` to express that the name is irrelevant.